.netlify/install.sh

```diff
@@ -7,3 +7,18 @@ V=0.16.0
 curl -LO https://github.com/bazelbuild/bazel/releases/download/$V/bazel-$V-installer-linux-x86_64.sh
 chmod +x bazel-$V-installer-linux-x86_64.sh
 ./bazel-$V-installer-linux-x86_64.sh --user
+
+# XXX: Hack to prevent the `gen_packages_list` rule from crashing
```

Comment:
If we rename `gen_packages_list` we should update this comment. Might also be worth explaining what about the situation causes it to crash if this hack isn't present.